gnucash/report/reports/standard/ifrs-cost-basis.scm

```diff
@@ -160,49 +160,55 @@ the split action field to detect capitalized fees on stock activity")
   (dividend-val get-dividend-val set-dividend-val!)
   (capgains-val get-capgains-val set-capgains-val!))
 
-;; "bitfield" Nabc a=neg b=zero c=pos
-(define (N001 x) (if (number? x) (>  x 0) #f))
-(define (N100 x) (if (number? x) (<  x 0) #f))
-(define (N010 x) (if (number? x) (=  x 0) #t))
-(define (N011 x) (if (number? x) (>= x 0) #t))
-(define (N110 x) (if (number? x) (<= x 0) #t))
-(define (N111 x) #t)
-;; N000 should be (not x) however we can accept a zero-amount split too
-(define (N000 x) (if (number? x) (=  x 0) #t))
+;; these functions test the various account values/amounts' signs
+(define (Dr>0 x) (if (number? x) (>  x 0) #f))
+(define (Cr>0 x) (if (number? x) (<  x 0) #f))
+(define (Dr≥0 x) (if (number? x) (>= x 0) #t))
+(define (Cr≥0 x) (if (number? x) (<= x 0) #t))
+(define (DrCr x) #t)
+(define (.... x) (if (number? x) (=  x 0) #t))
 
 ;;       <-- bitfield masks to identify ->  purchase?
 ;;       <---- the transaction type ----->  |  sale?
-;;       --stock-- cash cap  exp  divi capg |  |  narrative
-;;       amt  val       fees fees
 
 (define open-types
   (list
-   (list N001 N001 N100 N011 N000 N000 N000 #t #f "Open Long")
-   (list N100 N100 N001 N011 N000 N000 N000 #t #f "Open Short")))
+   ;;    --stock-- cash cap  exp  divi capg |  |   narrative
+   ;;    amt  val       fees fees
+   (list Dr>0 Dr>0 Cr>0 Dr≥0 .... .... .... #t #f "Open Long")
+   (list Cr>0 Cr>0 Dr>0 Dr≥0 .... .... .... #t #f "Open Short")))
```

Comment:
From my POV at least "Sell short" should be added to the glossary., but I am  open for better ideas.

Reply:
E. g. "A sale of commodities which is not covered by your current assets."

Reply:
+1 for adding it to the glossary. 

It's not a sale in the future, it's a sale now. The buy is in the future. I suggest something along the lines of "Selling borrowed units in the hope that when you buy them back later it will be at a lower price."

Reply:
Done as commit 01a7cbd.